src/coreclr/jit/lclvars.cpp

```diff
@@ -1690,6 +1747,7 @@ void Compiler::lvaClassifyParameterABI()
 
     for (unsigned lclNum = 0; lclNum < info.compArgsCount; lclNum++)
     {
+
```

Comment:
Nit: revert this unrelated whitespace change (mainly for my own sake to avoid conflicts)